src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -63,7 +63,7 @@ type Props = {
 type State = {
     view?: SidebarView,
     file?: BoxItem,
-    isVisible: boolean,
+    isVisible?: boolean,
```

Comment:
You can make this not be optional, and set it in the constructor

Reply:
Flow says I can't :(